README.md

```diff
@@ -1,36 +1,27 @@
-# Welcome to your [FASTN site](https://fastn.io/)
+Poppins: fastn Font Package
 
-FTD Font Package Template Repo
+This repository contains a [fastn font package](https://fpm.dev/featured/fonts/) containing [Google Font: 
+Poppins](https://fonts.google.com/specimen/Poppins/about).
 
+Geometric sans serif typefaces have been a popular design tool ever since these actors took to the world’s stage. Poppins is one of the new comers to this long tradition. With support for the Devanagari and Latin writing systems, it is an internationalist take on the genre.
 
-## 🚀 Project Structure
+Designers: Indian Type Foundry, Jonny Pinhorn
 
-Inside of your FASTN project, you'll see the following folders and files:
+## How To Use This Font In Your fastn Package:
 
-```
-/
-├── .build/
-│   └── index.html
-├── .packages/
-│   └── fifthtry.github.io/
-│   	└── All downloaded dependencies
-│   	
-├── FASTN.ftd
-├── index.ftd
-└── README.md
-```
+[Read the docs and demo](https://fastn-community.github.io/poppins-font
+/).
 
-## 🧞 Commands
-
-All commands are run from the root of the project, from a terminal:
-
-| Command                | Action                                             |
-| :--------------------- | :------------------------------------------------- |
-| `fastn build`            | FASTN builder installs all `FASTN` dependencies        |
-| `fastn serve`            | Starts local dev server at available port`localhost:8000`          |
+Also, check [playfair-display-font
+-typography](https://fastn-community.github.io/poppins-font
+-typography/)
 
+## 👀 Want to learn more?
 
+Feel free to check [our documentation](https://fpm.dev/) or jump into our [FifthTry Discord 
+server](https://discord.gg/bucrdvptYd).
 
-## 👀 Want to learn more?
+## License
 
-Feel free to check [our documentation](https://fastn.io/) or jump into our [FifthTry Discord server](https://discord.gg/bucrdvptYd).
+Since Arya Font is under [Open Font Licence](https://fonts.google.com/specimen/Playfair+Display/about), this fastn wrapper is also
```

Comment:
```suggestion
Since Poppins Font is under [Open Font Licence](https://fonts.google.com/specimen/Poppins/about?query=poppins), this fastn wrapper is also
```